src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlExceptionTest.cs

```diff
@@ -35,7 +33,6 @@ public void SerializationTest()
 
 #if NETFRAMEWORK
         [Fact]
-        [ActiveIssue("12161", TestPlatforms.AnyUnix)]
         public static void SqlExcpetionSerializationTest()
```

Comment:
```suggestion
        public static void SqlExceptionSerializationTest()
```

Since there are other typo corrections in the PR...